autopush/config.py

```diff
@@ -297,6 +299,9 @@ def from_argparse(cls, ns, **kwargs):
         if not ns.no_aws:
             ami_id = get_amid() or "Unknown"
 
+        import pdb;pdb.set_trace()
```

Comment:
woops

Reply:
also helps if I merge and push changes instead of wandering off to get a sandwich. ._.